moduleroot/.github/workflows/release.yml.erb

```diff
@@ -16,7 +16,13 @@ jobs:
           ruby-version: '2.7'
       - name: Build gem
         run: gem build *.gemspec
-      - name: Publish gem
+      - name: Publish gem to rubygems.org
         run: gem push *.gem
         env:
           GEM_HOST_API_KEY: '${{ secrets.RUBYGEMS_AUTH_TOKEN }}'
+      - name: Setup GitHub packages access
+        run: |
+          echo ":github: Bearer ${{ secrets.GITHUB_TOKEN }}" >> ~/.gem/credentials
+          chmod 0600 /home/runner/.gem/credentials
```

Comment:
Does this need to be done via a file? Wouldn't the `GEM_HOST_API_KEY` env var work?

Reply:
somehow that maps to the source thats used. I was hoping that an env var works, but `GEM_HOST_API_KEY` didn't work and the github docs don't mention any env var :(